streams/concat_streams.ts

```diff
@@ -1,78 +1,77 @@
 // Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
 
 /**
- * ConcatStreams extends the ReadableStreams class, concatenating any iterable
- * of ReadableStreams into a single ReadableStream.
+ * concatStreams is a function that concatenates any iterable of ReadableStreams
+ * into a single ReadableStream.
  *
  * @template T Type of the chunks in the ReadableStream.
  *
  * @example argument Usage
  * ```ts
- * import { ConcatStreams } from "@std/streams/concat-streams"
+ * import { concatStreams } from "@std/streams/concat-streams"
  *
```

Comment:
Please use `assertEquals()` and `Array.fromAsync()` in the example snippet ([guide](https://github.com/denoland/deno_std/blob/196101fbeabc3f40f0bc9db6510d9186b1111e6d/bytes/concat.ts#L10-L19)).